modules.json

```diff
@@ -483,6 +487,10 @@
     "commit": "7459cc2a5fd03bda902c2cd8d9d1fb387181b570",
     "path": "/nix/store/0jsq2yq7ds0szfhyrcsak99r79q7mf9c-replit-module-python-3.8"
   },
+  "python-3.8:v7-20231016-35990c4": {
+    "commit": "35990c496af3e4a137a67c1c7a5ebec0849610b3",
+    "path": "/nix/store/904k10vqg0avp4pg7fpzzma36p3jnw9v-replit-module-python-3.8"
+  },
```

Comment:
Why doesn't the python 3.10 module change?

Reply:
It must have already been using python310Packages.yapf which equates to nixpkgs.yapf.